Value Matching (#452)
openid/OpenID4VP: openid-4-verifiable-presentations-1_0.md

`values`:
: OPTIONAL. An array of strings, integers or boolean values that specifies the expected values of the claim.
If the `values` property is present, the Wallet SHOULD return the claim only if the
type and value of the claim both match for at least one of the elements in the array. Details of the processing
rules are defined in (#selecting_claims). Note that the expected behavior for value matching is only strict matching

> of the values, not any sort of fuzzy matching.

Reviewer: Im not sure whether we need this addition here, its pretty clear from the first sentence that the matching is exact.

Reviewer 2: Yeah agree, not sure this is needed. Paul's suggestion looks good

> type and value of the claim both match for at least one of the elements in the array. Details of the processing
> rules are defined in (#selecting_claims). Note that the expected behavior for value matching is only strict matching
> of the values, not any sort of fuzzy matching.

Reviewer 3: ```suggestion
type and value of the claim both match exactly for at least one of the elements in the array. Details of the processing
rules are defined in (#selecting_claims).
```

Reviewer 2: Yeah I prefer this.

exist in the Credential. Implementing this restriction may not be possible in
all cases, for example, if the Wallet does not have access to the claim value
before presentation or user consent or if another component routing

> the request to the Wallet does not have access to the claim value. It might also
> come down to a decision of the Wallet and/or the user if the value matching request
> is followed. Therefore, Verifiers must treat restrictions expressed using `values` as a
> best-effort way to improve user privacy, but MUST NOT rely on it for security checks.

Reviewer 4: it may ultimately be up to the Wallet and/or the user to decide whether th
```suggestion
the request to the Wallet does not have access to the claim value. It is ultimately up to the
Wallet and/or the End-user if the value matching request
is followed. Therefore, Verifiers MUST treat restrictions expressed using `values` as a
best-effort way to improve user privacy, but MUST NOT rely on it for security checks.
```
